apps/web/src/components/layout/components/v2/V3DashboardOptInWidget.tsx

```diff
@@ -0,0 +1,112 @@
+import { Card } from '@mantine/core';
+import { css } from '@novu/novui/css';
+import { Text, Title, Button, IconButton } from '@novu/novui';
+import { IconOutlineClose } from '@novu/novui/icons';
+import { useUser } from '@clerk/clerk-react';
+import { FeatureFlagsKeysEnum, V3DashboardOptInStatusEnum } from '@novu/shared';
+import { IS_SELF_HOSTED } from '../../../../config';
+import { useFeatureFlag } from '../../../../hooks';
+
+export function V3DashboardOptInWidget() {
```

Comment:
```suggestion
export function V2DashboardOptInWidget() {
```

We are not in V3 yet :p

Reply:
I kept it as V3 because all the figma designs are called "v3 dashboard" and we already use "V2" for something else. Imo this way its less confusing in code and communication.